layouts/joomla/form/field/email.php

```diff
@@ -69,6 +70,8 @@
     $required ? 'required' : '',
     $autofocus ? 'autofocus' : '',
     $dataAttribute,
+    // Force LTR input value in RTL
+    Factory::getLanguage()->isRtl() ? ' dir="ltr"' : '',
```

Comment:
```suggestion
    Factory::getLanguage()->isRtl() ? 'dir="ltr"' : '',
```
No need for the space as the array will be imploded with a space.

Reply:
got to love consistent coding :(